test/auth.js

```diff
@@ -146,3 +148,35 @@ test('Disabling an account works', function(t) {
         });
     });
 });
+
+test('First step of signin with OAuth works', function(t) {
+  t.plan(2);
+  t.throws(auth.signInWithOAuthRedirect, 'Should not work outside the browser');
+
+  var GOOD_OAUTH_LOGIN_DETAILS = {
+    appId: '24ad5676d5d25c6aa850dc5d5f63ec8c03dbc7ae113b6442b8571fce6c5b974c',
+    redirectUri: 'http://localhost',
```

Comment:
I couldn't get this to work in the browser. I also tried `location.href` and `"http://localhost:3735"`, but only got "The redirect uri included is not valid."